decidim-initiatives/app/types/decidim/initiatives/initiative_committee_member_type.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Initiatives
+    # This type represents a conference.
```

Comment:
:eye: this should represent an initiative

Reply:
indeed...